storage/object/api-cli/combining-iam-and-object-storage.mdx

```diff
@@ -0,0 +1,273 @@
+
+For example, if a user wants to list the objects in a bucket, they must ensure that:
+
+    - the IAM policy explicitly allows the user to list Objects (for example, with an `ObjectStorageReadOnly` or a `ObjectStorageObjectRead` permission set specified in the policy)
+    - if there is no existing bucket policy, the action allowed by IAM is enough to grant them list access
+    - if there is a bucket policy, the list action is explicitly allowed in the policy
+
+<Message type="important">
+  There are a couple of important things to keep in mind when using bucket and IAM policies concurrently:
+    - Even if you only want to define the permissions available for bucket policies, you must first define IAM policy permissions. This is a necessary step of the process.
+    - when you create a Scaleway API key with IAM, you are prompted to specify the API key’s [preferred Project for Object Storage](/identity-and-access-management/iam/api-cli/using-api-key-object-storage/). This API key will always use this Project when carrying out Object Storage actions via any API/CLI.
+    - the preferred Project must always be the one of the bucket where you wish to perform an action.
+</Message>
+
+<Message type="note">
+  Before the [arrival of IAM](/identity-and-access-management/iam/reference-content/migration/) principals in a Scaleway bucket policy were defined by the `project_id`. Now they can either be specified by the `user_id` or the `application_id`.
+</Message>
+
+## Examples
+
+In the sections below we provide different examples of how to combine IAM and bucket policies.
+
+<Message type="note">
+  In the following examples we use fictional UUIDs as the values for the `user_id`, `application_id`, `project_ids` and `organization_id` variables. Make sure you replace these UUIDs with your own when using the referenced policies.
+</Message>
+
+### Allowing all actions except DeleteObject
+
+In the example below, we provide the IAM and bucket policy configurations needed to allow an application to perform all actions in a bucket except to delete it.
+
+An IAM policy must be created to grant full access to Object Storage resources in Project 1 to application A.
+
+```json
+{
+   "name": "policy-S3-proj1",
+   "description": "This policy grants full access to Object Storage resources in Project 1 to application A",
+   "organization_id": "946552dd-a7dc-4d35-81df-11532f025b28",
+   "rules": [
+     {
+      "permission_set_names": [
+         "ObjectStorageFullAccess"
+      ],
+      "project_ids": [
+         "c6842bac-7938-4c04-9e03-f48147eee1f1"
+      ]
+     }
+   ],
+   "application_id": "950dde46-5cba-427d-a4f5-ce5a8a79717c"
+}
+```
+To block the deletion of objects in `production-bucket-1` for application A, the bucket policy below must be combined with the IAM policy above.
+```json
+{
+    "Version": "2012-10-17",
```

Comment:
The version should be updated here and for all Bucket policies using application_id or user_id

the correct version is : 
"Version":"2023-04-17"